src/test/java/SeleniumWebDriverTest.java

```diff
@@ -0,0 +1,47 @@
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class SeleniumWebDriverTest {
+        @Test
+        public void testDriver() throws InterruptedException {
+            System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+            WebDriver driver = new ChromeDriver();
+            driver.manage().window().maximize();
+            driver.get("https://www.selenium.dev");
+            Thread.sleep(3000);
+            driver.findElement(By.xpath("//a[@class='nav-item' and normalize-space()='Downloads']")).click();
+            String stableVersion = "3.141.59";
+            WebElement actualElement = driver.findElement(By.xpath("//a[@href=\"https://selenium-release.storage.googleapis.com/3.141/selenium-server-standalone-3.141.59.jar\" and text()='3.141.59']"));
```

Comment:
Here your code doesn't find latest stable version, it works by chance.
The selector strongly depends on @href value, which shouldn't be. You should select latest stable version, but not a link, which has text 3.141.59 and especially hardcoded value of @href attribute, because it can easily be in another place of a page and not to be the latest stable version, but just a link to download that particular version.
So you need to fix the selector.